src/baseInterceptor.js

```diff
@@ -128,39 +144,39 @@ function getRGBAname(arguments) {
   }
 }
 
-function getRGBname(arguments) {
-  const colorName = rgbColorName(arguments[0], arguments[1], arguments[2]);
-  const rgb = `(` + Math.round(arguments[0]) + `, ` + Math.round(arguments[1]) + `, ` + Math.round(arguments[2]) + `)`;
+function getRGBname(colArgs) {
+  const colorName = rgbColorName(colArgs[0], colArgs[1], colArgs[2]);
+  const rgb = '(' + Math.round(colArgs[0]) + ', ' + Math.round(colArgs[1]) + ', ' + Math.round(colArgs[2]) + ')';
   return ({
     'color': colorName,
     rgb
   });
 }
 
-function getHexname(arguments) {
-  let hex = arguments[0].slice(1);
-  if ((arguments[0].match(/\w/g)).length === 3) { // 3digithex
+function getHexname(colArgs) {
+  let hex = colArgs[0].slice(1);
+  if ((colArgs[0].match(/\w/g)).length === 3) { // 3digithex
     const h3x = hex.match(/\w/g)
-    hex = [h3x[0], h3x[0], h3x[1], h3x[1], h3x[2], h3x[2]].join(``);
+    hex = [h3x[0], h3x[0], h3x[1], h3x[1], h3x[2], h3x[2]].join('');
   }
   /* global hexColorName */
   const colorName = hexColorName(hex);
   const r = parseInt(hex[1] + hex[2], 16);
```

Comment:
@lm-n looks like this should be `hex[0] + hex[1]` and so on. Can you verify that? Right now hex value of `#800000` gives me an output of `dark red(0,0,0)` instead of 'dark red(128,0,0)

Reply:
fixed -.- sorry for that stupid mistake